tests-fuzz/src/generator/create_expr.rs

```diff
@@ -200,6 +206,116 @@ impl<R: Rng + 'static> Generator<CreateTableExpr, R> for CreateTableExprGenerato
     }
 }
 
+/// Generate a physical table with 2 columns: ts of TimestampType::Millisecond as time index and val of Float64Type.
+#[derive(Builder)]
+#[builder(pattern = "owned")]
+pub struct CreatePhysicalTableExprGenerator<R: Rng + 'static> {
+    #[builder(default = "Box::new(WordGenerator)")]
+    name_generator: Box<dyn Random<Ident, R>>,
+}
+
+impl<R: Rng + 'static> Generator<CreateTableExpr, R> for CreatePhysicalTableExprGenerator<R> {
+    type Error = Error;
+
+    fn generate(&self, rng: &mut R) -> Result<CreateTableExpr> {
+        let if_not_exists = rng.gen_bool(0.5);
```

Comment:
How about moving this to `CreatePhysicalTableExprGenerator`? Let the upper caller decide.